source/AI.cpp

```diff
@@ -455,7 +455,8 @@ void AI::UpdateEvents(const list<ShipEvent> &events)
 				// If you provoke the same ship twice, it should have an effect both times.
 				if(event.Type() & ShipEvent::PROVOKE)
 					newActions |= ShipEvent::PROVOKE;
-				event.TargetGovernment()->Offend(newActions, target->RequiredCrew());
+
```

Comment:
```suggestion
```

Reply:
not sure what you mean here, I gotta remove that ? why ?

Reply:
It's an unnecessary whitespace change.